Adding Endace to sumologic public partner

asimonsendace opened the pull request:
# PR Details

<!--- Provide a general summary of your changes in the Title above -->

## Description

<!--- Describe your changes in detail -->

## Related Issue

<!--- This project only accepts pull requests related to open issues -->
<!--- If suggesting a new feature or change, please discuss it in an issue first -->
<!--- If fixing a bug, there should be an issue describing it with steps to reproduce -->
<!--- Please link to the issue here: -->


## Checklist

<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->

- [ ] Changes follows the best practices from the given [docs](https://docs.google.com/document/d/1xfYfruru0RFWOH23GRrRzosSlepJUTcakLFHLHdpEtc/edit?usp=sharing).
- [ ] Updated CHANGELOG.md.
- [ ] Ran unit tests locally.
- [ ] Updated screenshots/sample logs if any.

ankitgoelcmu commented on Endace/Endace.json:
```diff
@@ -0,0 +1,922 @@
+                    "queries": [
+                        {
+                            "transient": false,
+                            "queryString": "_sourceCategory=suricata | count by src_ip | sort by _count | limit 10",
```
why `_sourceCategory` is hard coded every where ?

sumo-drew replied:
@himanshu219 - What is the recommended practice here? Does this need to be something else so that it's properly replaced with what the user specifies in Sumo at app install time? I thought our scripts in the repo handled this for us?

Is there something @asimonsendace needs to change here?

sumo-drew replied:
@asimonsendace - I see your email response further below. Typically what our partners do is let the user specific the sourceCategory at app install time, then in the instructions for installing the FERs, they call out that the sourceCategory must match what's set for the app. This is why @himanshu219 is asking.

himanshu219 replied:
yes no need to change, however the FERs too have hardcoded source category so we just have to document that whatever they provide while installing app should be same in the fer deploying script

ankitgoelcmu commented on Endace/README.md:
```diff
@@ -0,0 +1,69 @@
+
+The Endace App includes premade dashboards for logs such as Zeek, Suricata, Cisco ASA, Cisco Firepower and Palo Alto Networks, which include a Pivot-to-Vision link to connect you to your EndaceProbe for further investigation. You will need to download and run the python script from our Support Portal, which will create the Field Extraction Rules to automatically extract the needed fields and crea ...
+
+Python script: https://support.endace.com/s/eda-contentitem/a0MIS000000hfyE2AQ/endace-sumo-logic-python-script
```
I am not able to see this script; is this script created sumo logic collector as well as I don't see any step to create Sumo Logic collector to send data to ?

sumo-drew replied:
@asimonsendace - I think your response is ok here, @himanshu219 can we resolve this comment?

ankitgoelcmu commented on Endace/README.md:
```diff
@@ -0,0 +1,69 @@
+
+## How to use the App
+
+The Endace App includes premade dashboards for logs such as Zeek, Suricata, Cisco ASA, Cisco Firepower and Palo Alto Networks, which include a Pivot-to-Vision link to connect you to your EndaceProbe for further investigation. You will need to download and run the python script from our Support Portal, which will create the Field Extraction Rules to automatically extract the needed fields and crea ...
```
What kind of FERs will be created ? May be its good to document those

sumo-drew replied:
I think your response here is fine @asimonsendace. @himanshu219 if you agree, please resolve this comment.

asimonsendace commented:
Hi Ankit,

Please see my answers in red.

why _sourceCategory is hard coded every where ?

The _sourceCategory is hardcoded in the dashboards because they are dependent on the FERs that create the Pivot-to-Vision Link. The FERs are set to a specific source_category.

I am not able to see this script; is this script created sumo logic collector as well as I don't see any step to create Sumo Logic collector to send data to ?

In order to see the python script you will need to login to the Endace support page and then you should be able to download the python script. I have tested the link and was able to download the script. The script will not create the creation of the collectors, but I can add documentation on how the collector was setup - Keep in mind I did just use the default Linux collector install for this.

What kind of FERs will be created ? May be its good to document those

Because it's already noted what dashboards will be created and since the dashboards are dependent on each FER I don't believe it should be necessary.  The Endace App includes premade dashboards for logs such as Zeek, Suricata, Cisco ASA, Cisco Firepower and Palo Alto Networks,

Thanks,
Andrew Simons

From: Ankit Goel ***@***.***>
Sent: Tuesday, May 14, 2024 1:36 PM
To: SumoLogic/sumologic-public-partner-apps ***@***.***>
Cc: Andrew Simons ***@***.***>; Author ***@***.***>
Subject: Re: [SumoLogic/sumologic-public-partner-apps] Adding Endace to sumologic public partner (PR #70)


CAUTION: This email originated from outside the organisation. Do not click links or open attachments unless you recognise the sender and know the content is safe.

@ankitgoelcmu commented on this pull request.

________________________________

In Endace/Endace.json<https://github.com/SumoLogic/sumologic-public-partner-apps/pull/70#discussion_r1600482626>:

> +                    "description": "",

+                    "timeRange": null,

+                    "coloringRules": null,

+                    "linkedDashboards": []

+                },

+                {

+                    "id": null,

+                    "key": "panel34E577A187428840",

+                    "title": "Top 10 Source IPs",

+                    "visualSettings": "{\"version\":3,\"settings\":{\"table\":{\"version\":2,\"configuration\":{\"textTruncationMode\":\"none\",\"fontSize\":\"medium\",\"fontSizeOverride\":null,\"tableState\":null}},\"bar\":{\"chartType\":\"bar\",\"version\":2,\"configuration\":{\"colors\":{\"index\":null,\"overrides\":null},\"legend\":{\"enabled\":true,\"position\":\"right\",\"dimensions\":{\"width\":null,\"height\":null}},\"plotOptions\":{\"stacking\":null,\"lineToArea\":false},\"xAxis\":{\"title\":null,\"label\":{\"truncationMode\":\"middle\"},\"minorTickInterval\":null},\"yAxis\":[{\"title\":null,\"logScale\":false,\"min\":null,\"max\":null,\"bands\":null,\"minorTickInterval\":null}],\"seriesInfo\":{},\"showLineMarker\":true,\"lineMouseTracking\":true,\"multiSeriesTooltipOrder\":\"none\"}},\"column\":{\"chartType\":\"column\",\"version\":2,\"configuration\":{\"colors\":{\"index\":null,\"overrides\":null},\"legend\":{\"enabled\":true,\"position\":\"right\",\"dimensions\":{\"width\":null,\"height\":null}},\"plotOptions\":{\"stacking\":null,\"lineToArea\":false},\"xAxis\":{\"title\":null,\"label\":{\"truncationMode\":\"middle\"},\"minorTickInterval\":null},\"yAxis\":[{\"title\":null,\"logScale\":false,\"min\":null,\"max\":null,\"bands\":null,\"minorTickInterval\":null}],\"seriesInfo\":{},\"showLineMarker\":true,\"lineMouseTracking\":true,\"multiSeriesTooltipOrder\":\"none\"}},\"line\":{\"chartType\":\"line\",\"version\":2,\"configuration\":{\"colors\":{\"index\":null,\"overrides\":null},\"legend\":{\"enabled\":true,\"position\":\"right\",\"dimensions\":{\"width\":null,\"height\":null}},\"plotOptions\":{\"stacking\":null,\"lineToArea\":false},\"xAxis\":{\"title\":null,\"label\":{\"truncationMode\":\"middle\"},\"minorTickInterval\":null},\"yAxis\":[{\"title\":null,\"logScale\":false,\"min\":null,\"max\":null,\"bands\":null,\"minorTickInterval\":null}],\"seriesInfo\":{},\"showLineMarker\":true,\"lineMouseTracking\":true,\"multiSeriesTooltipOrder\":\"none\"}},\"area\":{\"chartType\":\"area\",\"version\":2,\"configuration\":{\"colors\":{\"index\":null,\"overrides\":null},\"legend\":{\"enabled\":true,\"position\":\"right\",\"dimensions\":{\"width\":null,\"height\":null}},\"plotOptions\":{\"stacking\":null,\"lineToArea\":false},\"xAxis\":{\"title\":null,\"label\":{\"truncationMode\":\"middle\"},\"minorTickInterval\":null},\"yAxis\":[{\"title\":null,\"logScale\":false,\"min\":null,\"max\":null,\"bands\":null,\"minorTickInterval\":null}],\"seriesInfo\":{},\"showLineMarker\":true,\"lineMouseTracking\":true,\"multiSeriesTooltipOrder\":\"none\"}},\"pie\":{\"chartType\":\"pie\",\"version\":3,\"configuration\":{\"donutMode\":true,\"colors\":{\"index\":null,\"overrides\":null},\"legend\":{\"enabled\":true,\"position\":\"right\"},\"plotOptions\":{\"label\":{\"truncationMode\":\"middle\"},\"showSliceBorders\":true,\"maxNumOfPieSlices\":\"10\"}}},\"boxplot\":{\"chartType\":\"boxplot\",\"version\":1,\"configuration\":{\"xAxis\":{\"title\":null,\"label\":{\"truncationMode\":\"middle\"}},\"yAxis\":{\"title\":null,\"min\":null,\"max\":null}}},\"map\":{\"chartType\":\"map\",\"version\":1,\"configuration\":{\"maptype\":\"cluster\"}},\"svv\":{\"version\":2,\"configuration\":{\"colorOverride\":null,\"backgroundColorEnabled\":false,\"noDataBehavior\":{\"enabled\":false,\"color\":\"#cccccc\",\"value\":null,\"isString\":null},\"valueType\":{\"number\":{\"showNumber\":true,\"unit\":null,\"colorsByValueRange\":null},\"boolean\":{\"trueColor\":\"#6aa84f\",\"falseColor\":\"#cc0000\"}},\"labels\":{\"prefix\":{\"enabled\":false,\"text\":\"\",\"align\":\"left\"},\"postfix\":{\"enabled\":false,\"text\":\"\",\"align\":\"right\"}}}},\"sankey\":{\"version\":1,\"configuration\":{}},\"text\":{\"version\":1,\"configuration\":{\"text\":null}},\"metrics\":{\"version\":2,\"configuration\":{\"blockSettings\":{},\"viewType\":\"timeline\",\"zoom\":\"xy\",\"yAxis\":{\"defaults\":[{\"name\":\"\"},{\"name\":\"\"}],\"custom\":[{},{}]},\"xAxis\":{\"scrubber\":true},\"outliers\":{\"enabled\":true,\"top\":1,\"scope\":\"Chart\",\"cyclicality\":false,\"threshold\":7,\"autoShowBand\":true},\"colorFamily\":\"METRIC_DEFAULT\"}},\"common\":{\"version\":1,\"configuration\":{\"drilldown\":{\"fallback\":{\"target\":{\"id\":null,\"name\":null,\"enabled\":null}}}}}},\"series\":{},\"general\":{\"type\":\"column\",\"displayType\":\"default\",\"roundDataPoints\":true,\"fillOpacity\":1,\"mode\":\"distribution\"},\"overrides\":[]}",

+                    "keepVisualSettingsConsistentWithParent": true,

+                    "panelType": "SumoSearchPanel",

+                    "queries": [

+                        {

+                            "transient": false,

+                            "queryString": "_sourceCategory=suricata | count by src_ip | sort by _count | limit 10",

why _sourceCategory is hard coded every where ?

________________________________

In Endace/README.md<https://github.com/SumoLogic/sumologic-public-partner-apps/pull/70#discussion_r1600484214>:

> +

+- [Introduction](#introduction)

+- [How to use the App](#How-to-use-the-App)

+- [Query Sample](#query-sample)

+- [Install the Sumo Logic App](#install-the-sumo-logic-app)

+- [Screenshots](#screenshots)

+

+## Introduction

+

+Bring clarity to every incident, alert, or issue with Endace packet capture evidence integrated directly into the Sumo Logic platform. Endace delivers scalable, always-on packet capture in on-premise and hybrid cloud environments for definitive network visibility.  Packets are the ultimate tamperproof source of truth that accelerates incident response for security threats, outages, and performance issues. This integration provides your operations team with fast access to the network packets (pcap) related to any incident or threat for rapid and precise incident response.

+

+## How to use the App

+

+The Endace App includes premade dashboards for logs such as Zeek, Suricata, Cisco ASA, Cisco Firepower and Palo Alto Networks, which include a Pivot-to-Vision link to connect you to your EndaceProbe for further investigation. You will need to download and run the python script from our Support Portal, which will create the Field Extraction Rules to automatically extract the needed fields and create the Pivot-to-Vision link.

+

+Python script: https://support.endace.com/s/eda-contentitem/a0MIS000000hfyE2AQ/endace-sumo-logic-python-script

I am not able to see this script; is this script created sumo logic collector as well as I don't see any step to create Sumo Logic collector to send data to ?

________________________________

In Endace/README.md<https://github.com/SumoLogic/sumologic-public-partner-apps/pull/70#discussion_r1600485140>:

> @@ -0,0 +1,69 @@

+## Endace App for Sumo Logic

+

+- [Introduction](#introduction)

+- [How to use the App](#How-to-use-the-App)

+- [Query Sample](#query-sample)

+- [Install the Sumo Logic App](#install-the-sumo-logic-app)

+- [Screenshots](#screenshots)

+

+## Introduction

+

+Bring clarity to every incident, alert, or issue with Endace packet capture evidence integrated directly into the Sumo Logic platform. Endace delivers scalable, always-on packet capture in on-premise and hybrid cloud environments for definitive network visibility.  Packets are the ultimate tamperproof source of truth that accelerates incident response for security threats, outages, and performance issues. This integration provides your operations team with fast access to the network packets (pcap) related to any incident or threat for rapid and precise incident response.

+

+## How to use the App

+

+The Endace App includes premade dashboards for logs such as Zeek, Suricata, Cisco ASA, Cisco Firepower and Palo Alto Networks, which include a Pivot-to-Vision link to connect you to your EndaceProbe for further investigation. You will need to download and run the python script from our Support Portal, which will create the Field Extraction Rules to automatically extract the needed fields and create the Pivot-to-Vision link.

What kind of FERs will be created ? May be its good to document those

-
Reply to this email directly, view it on GitHub<https://github.com/SumoLogic/sumologic-public-partner-apps/pull/70#pullrequestreview-2056133367>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BHR6Y5N2FMYSEE6FH42R3ILZCJKQJAVCNFSM6AAAAABGXAWBHKVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDANJWGEZTGMZWG4>.
You are receiving this because you authored the thread.Message ID: ***@***.******@***.***>>

sumo-drew commented:
Hi @ankitgoelcmu - Any updates or comments here for Endace?

himanshu219 commented:
@asimonsendace  
1. can we make that script public or as part of github repository ? So that if customers want to change the source category they can change it so we usually do not hard code source categories in our dashboards.
2. Also would be good to document what does the script does? what resources it creates
SumoLogic customers sometimes want to create separate source categories for their different environment or teams like Endace/Dev/Logs Endace/Prod/Logs
3. The icons and screenshots are structures inside resources folder. refer any other partner apps in master branch.
<img width="571" alt="image" src="https://github.com/SumoLogic/sumologic-public-partner-apps/assets/3620468/dbdd5331-4e47-410f-904f-a694e893b5c3">
4. Could you please add sample logs in resources/logs/ folder. This helps us in testing the app and also keeps a record of how older log format was.

asimonsendace commented:
Hi,


  1.  Is there a reason why we would want to add the python script in the github repo? If it is within the github repository then how the customers be able to access the script? From talking to Drew it was mentioned that the dashboards and FER creation needed to be separate as the App does not allow for FER creations.
  2.  I've added additional information in the README file for this
  3.  Updated
  4.  Updated

Thanks,
Andrew Simons

From: Himanshu Pal ***@***.***>
Sent: Wednesday, May 22, 2024 2:07 AM
To: SumoLogic/sumologic-public-partner-apps ***@***.***>
Cc: Andrew Simons ***@***.***>; Mention ***@***.***>
Subject: Re: [SumoLogic/sumologic-public-partner-apps] Adding Endace to sumologic public partner (PR #70)


CAUTION: This email originated from outside the organisation. Do not click links or open attachments unless you recognise the sender and know the content is safe.

@asimonsendace<https://github.com/asimonsendace>

  1.  can we make that script public or as part of github repository ? So that if customers want to change the source category they can change it so we usually do not hard code source categories in our dashboards.
  2.  Also would be good to document what does the script does? what resources it creates
SumoLogic customers sometimes want to create separate source categories for their different environment or teams like Endace/Dev/Logs Endace/Prod/Logs
  3.  The icons and screenshots are structures inside resources folder. refer any other partner apps in master branch.
image.png (view on web)<https://github.com/SumoLogic/sumologic-public-partner-apps/assets/3620468/dbdd5331-4e47-410f-904f-a694e893b5c3>

  1.  Could you please add sample logs in resources/logs/ folder. This helps us in testing the app and also keeps a record of how older log format was.

-
Reply to this email directly, view it on GitHub<https://github.com/SumoLogic/sumologic-public-partner-apps/pull/70#issuecomment-2124025257>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BHR6Y5JSFFDLHQGJ33E5AWDZDQ7XTAVCNFSM6AAAAABGXAWBHKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDCMRUGAZDKMRVG4>.
You are receiving this because you were mentioned.Message ID: ***@***.******@***.***>>

himanshu219 commented:
> Is there a reason why we would want to add the python script in the github repo? If it is within the github repository then how the customers be able to access the script? 
if it's in this repo then they can access it via github public link. 
Also in the Readme you can have a section similar to https://github.com/SumoLogic/sumologic-public-partner-apps/tree/master/DataminrPulse#set-up-collection so that customers clearly knows the order in which they have to create resources in sumo logic. 

Another pros of having FERs in this github repo is , we will soon be supporting FERs natively in the app (next-gen v2 app which you can currently see in our app catalog). So we can migrate this app to v2 if we know what the FERs are.

asimonsendace commented:
Hi,

Please find that I've updated the README file to point to the python script within github. Please let me know if you need anything further to approve the PR.

Thanks,
Andrew Simons

From: Himanshu Pal ***@***.***>
Sent: Friday, May 24, 2024 3:46 PM
To: SumoLogic/sumologic-public-partner-apps ***@***.***>
Cc: Andrew Simons ***@***.***>; Mention ***@***.***>
Subject: Re: [SumoLogic/sumologic-public-partner-apps] Adding Endace to sumologic public partner (PR #70)


CAUTION: This email originated from outside the organisation. Do not click links or open attachments unless you recognise the sender and know the content is safe.

Is there a reason why we would want to add the python script in the github repo? If it is within the github repository then how the customers be able to access the script?

if it's in this repo then they can access it via github public link.
Also in the Readme you can have a section similar to https://github.com/SumoLogic/sumologic-public-partner-apps/tree/master/DataminrPulse#set-up-collection so that customers clearly knows the order in which they have to create resources in sumo logic.

Another pros of having FERs in this github repo is , we will soon be supporting FERs natively in the app (next-gen v2 app which you can currently see in our app catalog). So we can migrate this app to v2 if we know what the FERs are.

-
Reply to this email directly, view it on GitHub<https://github.com/SumoLogic/sumologic-public-partner-apps/pull/70#issuecomment-2130328148>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BHR6Y5PPHRCXQFMABU673JDZD6RJTAVCNFSM6AAAAABGXAWBHKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDCMZQGMZDQMJUHA>.
You are receiving this because you were mentioned.Message ID: ***@***.******@***.***>>

himanshu219 commented on Endace/sumologic.py:
```diff
@@ -0,0 +1,110 @@
+access_key = '<access key>'
+
+# The API endpoint will depend on where your instance is located - in this case we are using US1. You can see more info at https://help.sumologic.com/docs/api/
+api_endpoint_post_extraction_rules = "https://api.sumologic.com/api/v1/extractionRules"
```
this url will change depending upon the deployment refer https://help.sumologic.com/docs/api/getting-started/

sumo-drew replied:
@asimonsendace - I would consider templating this like you did the access key so people don't ignore/miss this detail. Then just have them add it in. @himanshu219 - Would that be the approach you're looking for, or is the current approach sufficient?

himanshu219 replied:
yes correct

himanshu219 commented on Endace/README.md:
```diff
@@ -22,7 +22,7 @@ The Python script will create 5 different FERs:
 - Endace_Zeek
 
-Python script: https://support.endace.com/s/eda-contentitem/a0MIS000000hfyE2AQ/endace-sumo-logic-python-script
+Python script: ![Alt text](/sumologic.py?raw=true)
```
link is not rendering correctly it seems 
![sumologic.py](sumologic.py?raw=true)

himanshu219 commented on Endace/Endace.manifest.json:
```diff
@@ -0,0 +1,24 @@
+    ],
+    "communityURL": "",
+    "description": "Bring clarity to every incident, alert, or issue with Endace packet capture evidence integrated directly into the Sumo Logic platform. Endace delivers scalable, always-on packet capture in on-premise and hybrid cloud environments for definitive network visibility.  Packets are the ultimate tamperproof source of truth that accelerates incident response for security threats, out ...
+    "helpURL": "https://github.com/Endace/Integrations.SumoLogic/blob/main/README.md",
```
this helpurl is throwing 404 for me will it be live after we publish the app?

sumo-drew replied:
@asimonsendace - Can you confirm this is working?

asimonsendace commented:
Hi,

I'm not too sure what you are asking here - I've already noted in the comment above that "The API endpoint will depend on where your instance is located - in this case we are using US1."

Thanks,
Andrew Simons

From: Himanshu Pal ***@***.***>
Sent: Wednesday, May 29, 2024 11:30 PM
To: SumoLogic/sumologic-public-partner-apps ***@***.***>
Cc: Andrew Simons ***@***.***>; Mention ***@***.***>
Subject: Re: [SumoLogic/sumologic-public-partner-apps] Adding Endace to sumologic public partner (PR #70)


CAUTION: This email originated from outside the organisation. Do not click links or open attachments unless you recognise the sender and know the content is safe.

@himanshu219 commented on this pull request.

________________________________

In Endace/sumologic.py<https://github.com/SumoLogic/sumologic-public-partner-apps/pull/70#discussion_r1619861839>:

> +version = '1.0'

+

+# Variables for the Endace Probe URL

+hostname = '<probe hostname>'

+timezone = "etc/utc" # See link for list of time zones: https://en.wikipedia.org/wiki/List_of_tz_database_time_zones

+title = "Pivot%20from%20SumoLogic"

+datasources = 'tag:rotation-file'

+tools = 'trafficOverTime_by_app,conversations_by_ipaddress'

+url = f"https://{hostname}/vision2/pivotintovision/?title={title}&datasources={datasources}&incidenttime=TIME&reltime=5m&tools={tools}&ip_conv=SOURCEIP%26DESTIP<https://%7bhostname%7d/vision2/pivotintovision/?title=%7btitle%7d&datasources=%7bdatasources%7d&incidenttime=TIME&reltime=5m&tools=%7btools%7d&ip_conv=SOURCEIP%26DESTIP>"

+

+# This should be updated to your access id / key.

+access_id = '<access id>'

+access_key = '<access key>'

+

+# The API endpoint will depend on where your instance is located - in this case we are using US1. You can see more info at https://help.sumologic.com/docs/api/

+api_endpoint_post_extraction_rules = "https://api.sumologic.com/api/v1/extractionRules"

this url will change depending upon the deployment refer https://help.sumologic.com/docs/api/getting-started/

-
Reply to this email directly, view it on GitHub<https://github.com/SumoLogic/sumologic-public-partner-apps/pull/70#pullrequestreview-2087200971>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BHR6Y5M5B4RG2M6ZJCS4ZBTZE2TLBAVCNFSM6AAAAABGXAWBHKVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDAOBXGIYDAOJXGE>.
You are receiving this because you were mentioned.Message ID: ***@***.******@***.***>>

sumo-drew commented:
@asimonsendace - I agree putting the script in here is the right approach. They are indeed separate, but we can still include the python script in here. Thanks for doing that.

himanshu219 commented:
@asimonsendace  could you please add a section in the README on how to collect logs similar to https://github.com/SumoLogic/sumologic-public-partner-apps/tree/master/LambdaTest#collect-logs-for-lambdatest
or you can send me the link containing the steps  so I can just add it to the repo. Thanks!

asimonsendace commented:
Hi,

I've created another pull request adding on how to collect logs

https://github.com/SumoLogic/sumologic-public-partner-apps/pull/74

Thanks,
Andrew Simons

From: Himanshu Pal ***@***.***>
Sent: Friday, May 31, 2024 9:52 AM
To: SumoLogic/sumologic-public-partner-apps ***@***.***>
Cc: Andrew Simons ***@***.***>; Mention ***@***.***>
Subject: Re: [SumoLogic/sumologic-public-partner-apps] Adding Endace to sumologic public partner (PR #70)


CAUTION: This email originated from outside the organisation. Do not click links or open attachments unless you recognise the sender and know the content is safe.

@asimonsendace<https://github.com/asimonsendace> could you please add a section in the README on how to collect logs similar to https://github.com/SumoLogic/sumologic-public-partner-apps/tree/master/LambdaTest#collect-logs-for-lambdatest
or you can send me the link containing the steps so I can just add it to the repo. Thanks!

-
Reply to this email directly, view it on GitHub<https://github.com/SumoLogic/sumologic-public-partner-apps/pull/70#issuecomment-2142436658>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BHR6Y5JUM5FBSITVL2DW5OTZFCFAVAVCNFSM6AAAAABGXAWBHKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDCNBSGQZTMNRVHA>.
You are receiving this because you were mentioned.Message ID: ***@***.******@***.***>>

himanshu219 commented:
Thanks! have merged it.

himanshu219 commented:
@asimonsendace  I found couple of issues 
1. In panel "Top 10 Source IPs" of "Endace - Zeek"  dim-1.lab.endace.com is hardcoded and we are creating the Endace_Pivot_to_Vision again.
2. I ingested the sample logs but the three dashboards ciscoasa, palo alto and cisco firepower are not populating it seems the FER is not able to extract source field from json log. 
<img width="1155" alt="image" src="https://github.com/SumoLogic/sumologic-public-partner-apps/assets/3620468/a23cf093-aa7c-43b1-8777-acf395f0ddaf">

asimonsendace commented:
Hi,

Please be aware I created pull request https://github.com/SumoLogic/sumologic-public-partner-apps/pull/76 to correct the zeek dashboard.

For the source would it cause an issue to use "endace/messages" instead of just messages? It's really hard to troubleshoot just from the image. I'm happy to go on a call with you to help sort this out.

Thanks,
Andrew Simons

From: Himanshu Pal ***@***.***>
Sent: Saturday, June 8, 2024 10:44 AM
To: SumoLogic/sumologic-public-partner-apps ***@***.***>
Cc: Andrew Simons ***@***.***>; Mention ***@***.***>
Subject: Re: [SumoLogic/sumologic-public-partner-apps] Adding Endace to sumologic public partner (PR #70)


CAUTION: This email originated from outside the organisation. Do not click links or open attachments unless you recognise the sender and know the content is safe.

@asimonsendace<https://github.com/asimonsendace> I found couple of issues

  1.  In panel "Top 10 Source IPs" of "Endace - Zeek" dim-1.lab.endace.com is hardcoded and we are creating the Endace_Pivot_to_Vision again.
  2.  I ingested the sample logs but the three dashboards ciscoasa, palo alto and cisco firepower are not populating it seems the FER is not able to extract source field from json log.
image.png (view on web)<https://github.com/SumoLogic/sumologic-public-partner-apps/assets/3620468/a23cf093-aa7c-43b1-8777-acf395f0ddaf>

-
Reply to this email directly, view it on GitHub<https://github.com/SumoLogic/sumologic-public-partner-apps/pull/70#issuecomment-2156081296>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BHR6Y5L6H22FN4ETHWZEVXDZGMRC3AVCNFSM6AAAAABGXAWBHKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDCNJWGA4DCMRZGY>.
You are receiving this because you were mentioned.Message ID: ***@***.******@***.***>>
